Rmarkdown/Multiple_Ascending_Dose_PKPD_continuous.Rmd

```diff
@@ -13,70 +13,88 @@ This document contains exploratory plots for continuous PD data as well as the R
 
 ## Setup
 
-```{r, echo = TRUE, warning=FALSE, message=FALSE}
-# remove reference to home directory in libPaths
-.libPaths(grep("home", .libPaths(), value=TRUE, invert=TRUE))
-
-# add localLib to libPaths for locally installed packages
-.libPaths(c("localLib", .libPaths()))
-
-# will load from first filepath first, then look in .libPaths for more packages not in first path
-# version matches package in first filepath, in the case of multiple instances of a package
-
-# library(rmarkdown)
-library(gridExtra)
-library(grid)
+```{r, warning=FALSE, message=FALSE}
 library(ggplot2)
 library(dplyr)
-library(RxODE)
-library(caTools)
-
-theme_set(theme_bw(base_size=12))
+library(tidyr)
+library(xgxr, lib.loc = "../Rlib")
```

Comment:
We can remove the ', lib.loc = "../Rlib"' now that the xgxr package is installed